tensorflow-framework/src/test/java/org/tensorflow/framework/activations/SwishTest.java

```diff
@@ -0,0 +1,106 @@
+/* Copyright 2020 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+=======================================================================*/
+package org.tensorflow.framework.activations;
+
+import org.junit.jupiter.api.*;
+import org.tensorflow.Operand;
+import org.tensorflow.framework.utils.TestSession;
+import org.tensorflow.op.Ops;
+import org.tensorflow.types.TFloat32;
+import org.tensorflow.types.TFloat64;
+import org.tensorflow.types.TInt32;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+/** @author Jim Clarke */
+public class SwishTest {
+  private final TestSession.Mode[] tfModes = {TestSession.Mode.EAGER, TestSession.Mode.GRAPH};
+
+  public SwishTest() {}
+
+  @BeforeAll
+  public static void setUpClass() {}
+
+  @AfterAll
+  public static void tearDownClass() {}
+
+  @BeforeEach
+  public void setUp() {}
+
+  @AfterEach
+  public void tearDown() {}
+
+  /** Test of Swish call method */
+  @Test
+  public void testCallInt() {
+    int[] input = {1, -2, 3, -4, -1, 2, -3, 4};
+    for (TestSession.Mode tfMode : tfModes)
+      assertThrows(
+          java.lang.IllegalArgumentException.class,
+          () -> {
+            try (TestSession session = TestSession.createTestSession(tfMode)) {
+              Ops tf = session.getTF();
+              Swish<TInt32> instance = new Swish<>(tf);
+              Operand<TInt32> result = instance.call(tf.constant(input));
+            }
+          });
+  }
+
+  /** Test of Swish call method */
+  @Test
+  public void testCallFloat() {
+    float[] input = {1, -2, 3, -4, -5, 6, -7, 8};
+    float[] expected = {
```

Comment:
Where were the expected values computed? I would've thought we'd be using the examples provided in the Python API docs: https://www.tensorflow.org/api_docs/python/tf/keras/activations/swish

Reply:
For activations, the expected values where determined by running the same inputs against Python TF.